### ISSUE TITLE
Backport default page size improvements to 0.3.X

### ISSUE THREAD

**Issue by TylerStaplerAtFanatics** [Twingate/terraform-provider-twingate#402]
tl;dr: Can you backport a few important changes to the 0.3.X version of the provider?

We have a quite a few terraform resources which require 0.3.X to use.

Because of the breaking changes in 1.0.0 it's difficult for us to migrate our existing resources to use the newer provider versions.

We consistently see errors that Twingate support has attributed to rate limiting:
I believe the fixes are in this commit https://github.com/Twingate/terraform-provider-twingate/commit/e7be310bd815a2b7ef55f20b05dd2a097fd0c0d2
```
Error: failed to read service account with id All: Post "https://betfanatics.twingate.com/api/graphql/": POST https://betfanatics.twingate.com/api/graphql/ giving up after 11 attempt(s)

  with module.infrastructure.data.twingate_service_accounts.database_scripts,
  on /tmp/terraform-data-dir/modules/infrastructure/root/sportsbook-infra-services/main.tf line 74, in data "twingate_service_accounts" "database_scripts":
  74: data "twingate_service_accounts" "database_scripts" {


Error: failed to read service account with id <REDACTED>: Post "https://betfanatics.twingate.com/api/graphql/": POST https://betfanatics.twingate.com/api/graphql/ giving up after 11 attempt(s)

  with module.infrastructure.twingate_service_account.retail_kiosk,
  on /tmp/terraform-data-dir/modules/infrastructure/root/sportsbook-infra-services/main.tf line 78, in resource "twingate_service_account" "retail_kiosk":
  78: resource "twingate_service_account" "retail_kiosk" {


Error: failed to read service account with id <REDACTED>: Post "https://betfanatics.twingate.com/api/graphql/": POST https://betfanatics.twingate.com/api/graphql/ giving up after 11 attempt(s)

  with module.infrastructure.twingate_service_account.retail_awx,
  on /tmp/terraform-data-dir/modules/infrastructure/root/sportsbook-infra-services/main.tf line 83, in resource "twingate_service_account" "retail_awx":
  83: resource "twingate_service_account" "retail_awx" {
```

More recently we have seen 400 errors from a cursor:
```
Error: failed to read group with id <REDACTED>: Message: 400 Bad Request; body: "{\"errors\":[{\"message\":\"Variable '$usersEndCursor' is never used in operation 'readGroup'.\",\"locations\":[{\"line\":1,\"column\":24}]}]}", Locations: [], Extensions: map[code:request_error]

  with module.infrastructure.data.twingate_group.users_group,
  on /tmp/terraform-data-dir/modules/infrastructure/root/sportsbook-infra-services/main.tf line 57, in data "twingate_group" "users_group":
  57: data "twingate_group" "users_group" {


Error: failed to read group with id <REDACTED>: Message: 400 Bad Request; body: "{\"errors\":[{\"message\":\"Variable '$usersEndCursor' is never used in operation 'readGroup'.\",\"locations\":[{\"line\":1,\"column\":24}]}]}", Locations: [], Extensions: map[code:request_error]

  with module.infrastructure.data.twingate_group.backoffice_group,
  on /tmp/terraform-data-dir/modules/infrastructure/root/sportsbook-infra-services/main.tf line 67, in data "twingate_group" "backoffice_group":
  67: data "twingate_group" "backoffice_group" {
```

Are you open to backporting the fixes for these changes? I'm happy to contribute a PR if you point me in the right direction.

**Comment by alexb-twingate** [comment]
Hi @TylerStaplerAtFanatics - we're looking into this. We'll be able to fix the 400 errors that you've been seeing, but we won't be able to fix the rate limiting issues via the provider. We're not able to backport the PR you tagged, but we can increase your read limit if that would help.

**Comment by ekampf** [comment]
Hey @TylerStaplerAtFanatics 
Just released `v0.3.4` which should fix the 400 error issue you're seeing
